### PR TITLE
Manual bindings for RTSPToken

### PR DESCRIPTION
**Pull request by MathieuDuponchelle**
I tested it and it works, had to fumble a bit with the `GstRcExt` part, not entirely sure about it but I was getting `Can not impl trait from another crate for Vec<T> with T from this crate` and this seemed to be the correct way to deal with it?

### CODE REVIEW

File: gstreamer-rtsp-server/src/r_t_s_p_token.rs [hunk 1]
```diff
@@ -1,12 +1,33 @@
...
+use glib::value::ToSendValue;
 use glib::translate::*;
 
-impl RTSPToken {
-    pub fn new(values: &[(&str, &ToSendValue)]) -> RTSPToken {
-        let mut token = RTSPToken::new_empty();
+use gst::miniobject::{MiniObject, GstRc};
+
+pub trait GstRcExt<T: MiniObject> {
+    fn new_empty() -> Self;
```
Comment (sdroege) [inline gstreamer-rtsp-server/src/r_t_s_p_token.rs]: Hrm this is quite ugly! So there's a problem implementing miniobjects outside the main crate.

With your solution, can you call `RTSPToken::new_empty()`? The name should probably be `GstRcRTSPTokenExt` though, we would need the same thing for other external miniobjects and traits with the same name, while not conflicting, cause weird issues sometimes.

File: gstreamer-rtsp-server/src/auto/mod.rs [hunk 2]
```diff
@@ -61,9 +61,6 @@ pub use self::r_t_s_p_thread_pool::RTSPThreadPoolExt;
 mod r_t_s_p_address;
 pub use self::r_t_s_p_address::RTSPAddress;
 
-mod r_t_s_p_token;
-pub use self::r_t_s_p_token::RTSPToken;
```
[the remark this answers is not in the document]
Comment (MathieuDuponchelle) [reply gstreamer-rtsp-server/src/auto/mod.rs]: We already had an override for RTSPToken, and were importing it with:

```
mod r_t_s_p_token;
pub use r_t_s_p_token::*;
```

I assumed that was enough?

File: gstreamer-rtsp-server/src/r_t_s_p_token.rs (hunk 1 above)
Comment (MathieuDuponchelle) [reply gstreamer-rtsp-server/src/r_t_s_p_token.rs]: Done

File: gstreamer-rtsp-server/src/auto/mod.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (MathieuDuponchelle) [reply gstreamer-rtsp-server/src/auto/mod.rs]: Done

**Pull request merged**